.scalafmt.conf

```diff
@@ -0,0 +1,13 @@
+style = defaultWithAlign
+
+align                      = some
+align.tokens               = [off]
+danglingParentheses        = true
+docstrings                 = JavaDoc
+indentOperator             = spray
+maxColumn                  = 120
+rewrite.rules              = [RedundantParens, SortImports]
```

Comment:
You seem to prefer `AvoidInfix` as well, but we may not want to rewrite those.

Reply:
My strong pref is for `RedundantBraces` as well but...

Reply:
> RedundantBraces

I think that depends how large the statement is. If it's a real one-liner then yes, but if it's wrapping over several lines and having comments I prefer some extra `{ }` to clarify the scope.

Reply:
I would like to use `AvoidInfix` but it is not nice for some things, like
```
withAttributes(traversalBuilder.attributes.and(attr))
```

but maybe the total result is better? I have pushed that as a separate commit so we can compare and decide: 04ae484

Reply:
There were some more trouble with AvoidInfix that I have solved in 32199ce and 1627f14

Reply:
`AvoidInfix` seems good now, WDYT?

Reply:
`rewrite.neverInfix.excludeFilters` worked great, now it's `AvoidInfix` with some listed exceptions

Reply:
"some" :D